src/repositories/tasks.ts

```diff
@@ -0,0 +1,41 @@
+import { OAuth1Client } from '@extrahorizon/javascript-sdk';
+import { permissionModes } from '../commands/tasks/taskConfig';
+
+export interface FunctionCreation {
+  name: string;
+  description: string;
+  entryPoint: string;
+  code: string;
+  runtime: string;
+  timeLimit?: number;
+  memoryLimit?: number;
+  environmentVariables?: Record<string, any>;
+  executionOptions?: {
+    permissionMode: permissionModes;
+  };
+  retryPolicy?: {
+    enabled: boolean;
+    errorsToRetry: string[];
+  };
+}
+
+export const functions = ({
```

Comment:
(silly, no need to change) Why not simply calling this the `functionsRepository`?

Reply:
Hmmm, because functions are embedded in the tasks interfaces in the SDK so it made more sense to me to follow the same structure here 🤔 I'm open to changing it if it's the consensus...

Reply:
No need to to change! I my opinion the structure of the SDK didn't need to "dictate" what the structure of the CLI uses, even despite the SDK being managed by us. And I still think the structure of the SDK makes sense for the SDK itself though